source/renderer/app/components/wallet/settings/WalletPublicKeyDialog.scss

```diff
@@ -1,5 +1,13 @@
 @import '../../../themes/mixins/error-message';
 
+.dialog {
+  :global {
+    .Dialog_contentWrapper::-webkit-scrollbar-track {
```

Comment:
@aleksandardjordjeviciohk shouldn't we put this into an override for the Dialog instead of the global here? 🤔

Reply:
This is obsolete now, as it was a leftover from previous implementation. Deleted now.